assets/javascripts/discourse/components/tag-drop-assigned.js.es6

```diff
@@ -0,0 +1,22 @@
+import TagDrop from "select-kit/components/tag-drop";
+
+export const NO_TAG_ID = "no-tags";
+export const ALL_TAGS_ID = "all-tags";
+export const NONE_TAG_ID = "none";
+
+export default TagDrop.extend({
+  actions: {
+    onChange(tagId) {
+      switch (tagId) {
```

Comment:
I think the `if` / `else if` / `else` approach like you did for the categories is a tiny bit easier on the eyes 😉

Reply:
Sure, ill make that change 😃